sources/ca/ns/province.json

```diff
@@ -13,13 +13,16 @@
         "addresses": [
             {
                 "name": "state",
-                "data": "https://nsgiwa.novascotia.ca/arcgis/rest/services/BASE/BASE_NS_CivicAddress_File_WM84/MapServer/0",
+                "data": "https://data.novascotia.ca/api/geospatial/tntn-er5g?method=export&format=Shapefile",
```

Comment:
The old URL seems to exist still. Why did we change this URL? I'd prefer to use Esri endpoints as they're usually kept up to date.